com.unity.render-pipelines.universal/Editor/Converter/RenderPipelineConvertersEditor.cs

```diff
@@ -120,18 +121,25 @@ void OnEnable()
 
             // This is the drop down choices.
             m_ConverterContainers = TypeCache.GetTypesDerivedFrom<RenderPipelineConverterContainer>();
+            var converterList = TypeCache.GetTypesDerivedFrom<RenderPipelineConverter>();
 
-            var converters = TypeCache.GetTypesDerivedFrom<RenderPipelineConverter>();
-            for (int i = 0; i < converters.Count; ++i)
+            for (int i = 0; i < converterList.Count; ++i)
             {
                 // Iterate over the converters
-                RenderPipelineConverter conv = (RenderPipelineConverter)Activator.CreateInstance(converters[i]);
+                RenderPipelineConverter conv = (RenderPipelineConverter)Activator.CreateInstance(converterList[i]);
                 m_CoreConvertersList.Add(conv);
+            }
+
+            // this need to be sorted by Priority property
+            m_CoreConvertersList = m_CoreConvertersList
+                .OrderBy(o => o.GetType().GetProperty("priority").GetValue(o, null)).ToList();
```

Comment:
Perhaps we can get rid of the reflection here and on line 129 by using an interface.

Reply:
done